apps/vanilla/src/pages/blackjack/index.ts

```diff
@@ -0,0 +1,67 @@
+import { Debugger } from './classes/debugger';
+import { DiceGenerator } from './classes/dice-generator';
+import { Player } from './classes/player';
+import { ResultDisplayer } from './classes/result-displayer';
+import { WinnerDisplayer } from './classes/winner-displayer';
+import { TurnGenerator } from './classes/turn-generator';
+import { NUMBER_OF_PLAYER } from './constants';
+
+/** Initialize Game.*/
+class App {
```

Comment:
```suggestion
class Game {
```
Looks like this class deserves its own file